File: ClimbingStairs/simple.java
class Solution {
public:
    int climbStairs(int n) {
        if(n == 0 || n == 1)
        {
            return n;
        }
        int a = 1;
        int b = 1;
        for(int i = 1; i < n; i++)
        {
            if(i%2 == 0)
            {
                b += a;
            }else
            {
                a += b;
            }
        }
        if(n%2 == 0)
        {
            return a;
        }else
        {
            return b;
        }
    }
};

File: BinaryTreeLevelOrderTraversalII/simple.py
class Solution:
    # @param {TreeNode} root
    # @return {integer[][]}
    def levelOrderBottom(self, root):
        if root is None: return []
        levels = [[root]]
        results = []

        while levels:
            level = levels.pop()

            result_row = [node.val for node in level]
            results.insert(0, result_row)

            next_level = []
            for node in level:
                if node.left:
                    next_level.append(node.left)
                if node.right:
                    next_level.append(node.right)

            if len(next_level) > 0:
                levels.append(next_level)

        return results
